### ISSUE TITLE
absorption example error

### ISSUE THREAD

**Issue by pawel21** [cosimoNigro/agnpy#38]
Hi,

I have tried use absorption example [doc](https://agnpy.readthedocs.io/en/latest/absorption.html)
and I got the following error:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-b07779d8d11b> in <module>
      2 r = 1.1e16 * u.cm
      3 
----> 4 absorption_disk = Absorption(blob, disk, r=r)
      5 absorption_blr = Absorption(blob, blr, r=r)
      6 absorption_dt = Absorption(blob, dt, r=r)

~/anaconda3/envs/agn/lib/python3.7/site-packages/agnpy-0.0.7-py3.7.egg/agnpy/absorption.py in __init__(self, blob, target, r)
     43         self.target = target
     44         self.r = r
---> 45         self.set_mu()
     46         self.set_phi()
     47         self.set_l()

~/anaconda3/envs/agn/lib/python3.7/site-packages/agnpy-0.0.7-py3.7.egg/agnpy/absorption.py in set_mu(self, mu_size)
     49     def set_mu(self, mu_size=100):
     50         self.mu_size = mu_size
---> 51         if self.target.type == "SSDisk":
     52             # in case of hte disk the mu interval does not go from -1 to 1
     53             r_tilde = (self.r / self.target.R_g).to_value("")

AttributeError: 'SSDisk' object has no attribute 'type'
```
I guess this is due to disk not have attribute 'type', but 'name'.
Could you reploduce my error?

**Comment by cosimoNigro** [comment]
Hi @pawel21,

thanks for reporting. You are exactly right, I changed the targets but forgot to update the `Absorption` module.
I planned to do it in a second moment since I had some issues to fix related to that module (e.g. Issue #16).
Will take a look this week.

Cheers!

**Comment by cosimoNigro** [comment]
You are also very welcome to fix it yourself, if you feel you have time.
The fix on #16 requires a bit of thought but the one is preventing you to run the tutorial does not ;)

**Comment by pawel21** [comment]
Thank you, I will fix it.

**Comment by cosimoNigro** [comment]
Solved by PR #39 
thanks @pawel21 !
